lib/config/validator.js

```diff
@@ -75,6 +77,14 @@ class Validator {
             logger.warning(`Passing "${this.webpackConfig.sharedCommonsEntryName}" to createSharedEntry() is not recommended, as it will override the built-in cache group by this name.`);
         }
     }
+
+    _validateCacheGroupNames() {
+        for (const groupName of Object.keys(this.webpackConfig.cacheGroups)) {
+            if (['vendors', 'default'].includes(groupName)) {
```

Comment:
I think it might be called `defaultVendors` internally? https://github.com/webpack/webpack/blob/f5e69fcc9b4142aadf8588e96f5e80d6b7cc5320/lib/config/defaults.js#L563-L574

Reply:
Hmm, I put `vendors` because we currently have a validation on `createSharedEntry()` including that group: https://github.com/symfony/webpack-encore/blob/bdf8ba46cc447c84ed2f4daf80c16122be191189/lib/config/validator.js#L73-L77

But you seem to be correct :)